models/intermediate/int_jira__issue_join.sql

```diff
@@ -61,13 +65,22 @@ join_issue as (
 
         priority.priority_name as current_priority,
 
+        {% if var('using_sprints', True) %}
         issue_sprint.sprint_id,
         issue_sprint.sprint_name,
         issue_sprint.count_sprint_changes,
-
+        issue_sprint.sprint_started_at,
+        issue_sprint.sprint_ended_at,
+        issue_sprint.sprint_completed_at,
+        issue_sprint.sprint_started_at <= {{ dbt_utils.current_timestamp() }}
+          and issue_sprint.sprint_ended_at >= {{ dbt_utils.current_timestamp() }} as is_active_sprint,
```

Comment:
if an issue is slotted into a sprint that hasn't started yet, this will be `null` -- do we want to coalesce this to `false` because it's technically inactive?

also, Jira considers a formerly-active sprint to be inactive after it has been marked as *completed*, rather than when the projected end date has passed. so if i start a sprint that's supposed to end in 2 weeks, but i mark it as completed today it will not appear in the "active sprints" tab in that project (conversely, if i don't mark it complete after its end date as passed, it remains in "active sprints"). perhaps we might wanna use `completed_at` instead of `ended_at` to determine activity? i suppose it could depend on how folks use jira, so i'm open to opinions!

Reply:
I've updated the logic to mirror what the logic Jira uses. Can you please review for accuracy?